apps/communications/contacts/views/list/list.html

```diff
@@ -0,0 +1,189 @@
+    <link rel="localization" href="/shared/locales/date/date.{locale}.properties">
+    <link rel="localization" href="/shared/locales/phone_types/phone_types.{locale}.properties">
+
+    <!--
+      <script defer src="/shared/js/contact_photo_helper.js"></script>
+      <script defer src="/shared/js/contacts/utilities/ice_store.js"></script>
+      <script defer src="/shared/js/contacts/utilities/image_loader.js"></script>
+      <script defer src="/shared/js/tag_visibility_monitor.js"></script>
+      <script defer src="/shared/js/component_utils.js"></script>
+      <script defer src="/shared/js/contacts/utilities/templates.js"></script>
+      <script defer src="/shared/js/contacts/contacts_shortcuts.js"></script>
+      <script defer src="/shared/js/text_normalizer.js"></script>
+      <script defer src="/shared/js/contacts/import/utilities/status.js"></script>
+      <script defer src="/shared/js/contacts/utilities/dom.js"></script>
+      <script defer src="/shared/js/confirm.js"></script>
+      <script defer src="/contacts/js/utilities/cookie.js"></script>
+      <script defer src="/shared/js/async_storage.js"></script>
+      <script defer src="/shared/js/contacts/import/utilities/config.js"></script>
+      <script defer src="/shared/js/contacts/search.js"></script>
+    -->
+
+    <script defer src="/shared/js/lazy_loader.js"></script>
+    <script defer src="/contacts/js/utilities/performance_helper.js"></script>
+    <script defer src="/contacts/js/cache.js"></script>
+    <script defer src='/contacts/views/list/js/boot.js'></script>
+
+    <meta name="defaultLanguage" content="en-US">
+    <meta name="availableLanguages" content="en-US">
+    <link rel="localization" href="/contacts/locales/contacts.{locale}.properties">
+    <link rel="localization" href="/shared/locales/import_contacts/import_contacts.{locale}.properties">
+    <link rel="localization" href="/shared/locales/date/date.{locale}.properties">
+    <link rel="localization" href="/shared/locales/phone_types/phone_types.{locale}.properties">
+
+    <link rel="import" href="/contacts/elements/confirm.html">
+    <link rel="import" href="/contacts/elements/search.html">
+    <link rel="import" href="/contacts/elements/status.html">
+  </head>
+
+  <body role="application" class="app-contacts skin-comms theme-communications" data-action="list">
+
+      <!-- Contacts List Section -->
+      <section id='view-contacts-list' role="region" data-state="active" class="view view-noscroll view-contacts-list">
+        <gaia-header id="contacts-list-header">
+          <h1 id="app-title" data-l10n-id="contacts"></h1>
+          <a id="add-contact-button" href="/contacts/views/form/form.html?action=new" data-icon="add" data-l10n-id="addButton"></a>
+          <a id="settings-button" href="/contacts/views/settings/settings.html" data-icon="settings" data-l10n-id="settingsButton"></a>
```

Comment:
As agreed this is part of the Settings PR, so let's review everything but this, and we will review the whole solution in a separate branch.
If everything works as expected, we will land both commits into NGA branch, and the app will be working under NGA approach.